### PR TITLE
Simplify celery images

### PR DESCRIPTION
**Pull request by Kelvin-M**
As https://github.com/SubstraFoundation/substra-backend/pull/320 has been merged, we can simplify celery services images to only one.

@AlexandrePicosson  we may need to update docker-hub to take this into account

Companion PR
https://github.com/SubstraFoundation/substra-tests/pull/146

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Backend of the Substra platform
 
 |  substra-backend | celeryworker | celerybeat | flower |
```
Comment (AurelienGasser) [inline README.md:5]: you fogot this line

**Review by AurelienGasser — CHANGES_REQUESTED** [review]
Just a minor oversight in the README

File: README.md (hunk 1 above)
Comment (Kelvin-M) [reply README.md]: Fixed, thank you !

**Review by AurelienGasser — APPROVED** [review]

**Comment by AurelienGasser** [comment]
@Kelvin-M Have you considered bumping the chart?

**Review by AlexandrePicosson — APPROVED** [review]
Sure I will create the repository on DockerHub, just let me know when this gets merged.

Maybe we should also define a time for when we will retire the old images, is there a way to manage EOL for versions somewhere for example ?
Or do we keep the old images/repository for an infinite amount of time ?

**Comment by Kelvin-M** [comment]
@AlexandrePicosson we may need to create it before this branch is merged no ?

For old images, I don't know. We may keep it like this and when we won't support them anymore remove them (maybe in 2021) ?

**Comment by AlexandrePicosson** [comment]
It does not matter, when creating the repo we can trigger a manual build so it should be okay, I'll look into this now.

I have no strong opinion on the subject, I just wanted to know if there is already a policy for this kind of situations. Just deleting it when we drop the support of the last version is fine for me.

**Comment by AlexandrePicosson** [comment]
I created the DockerHub repository here : https://hub.docker.com/repository/docker/substrafoundation/celery

**Comment by AurelienGasser** [comment]
@Kelvin-M I think you forgot to update the `chart/CHANGELOG.md`

**Review by AlexandrePicosson — APPROVED** [review]

**Review by inalgnu — APPROVED** [review]

**Pull request merged**